docs/advanced/gpu.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Edge Pods use GPU
+sidebar_position: 9
+---
+
+## Abstract
+
+With the development of edge AI, the demand for deploying GPU applications on edge nodes is gradually increasing. Currently, KubeEdge can manage GPU nodes through some configurations, 
+and allocate GPU resources to user edge applications through the k8s device-plugin component. If you need to use this feature, please refer to the steps below.
+
+## Getting Started
+
+### GPU running environment construction
```

Comment:
Can we instead just point some URLs such as https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html to make sure Nvidia GPU is enabled and configured for container runtime?
This is obviously not the procedure KubeEdge provides, to avoid having the maintenance cost to catch up with Nvidia specific procedure, it would be easier for the doc maintainers?